### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by belfie13** [helge17/tuxguitar#239] [General]
Who did the tuxguitar icon? any source files i can play around with for fun? I want to do some different guitars with the penguin. not a great topic but I'd have fun with it :D

**Comment by helge17** [comment]
It would be very cool to have an updated icon, see issue #138

The original TuxGuitar icons are quite old, I think Julian made them. I can't find a source file (SVG etc.) for them. The versions with the best resolution should be

- https://github.com/helge17/tuxguitar/blob/master/website/logo.png
- https://github.com/helge17/tuxguitar/blob/master/desktop/build-scripts/common-resources/common-windows-installer-icons/TG_logo_268x268.png

There is also an SVG icon in the abandoned archive 2.0beta, but it's not that sophisticated:

- https://github.com/helge17/tuxguitar-2.0beta/blob/master/TuxGuitar/icon-src/Symbolic/icon.svg

**Comment by belfie13** [comment]
aha kool thanks, i couldn't find those, it'll help.

**Comment by guiv42** [comment]
I don't know where it comes from, but I really like the icon on [wikipedia page](https://en.m.wikipedia.org/wiki/TuxGuitar)

**Reply by helge17** [reply]
This icon is from the [blue_serious](https://github.com/helge17/tuxguitar/blob/master/desktop/TuxGuitar/share/skins/blue_serious/skin.svg) skin.

**Comment by lborl** [comment]
I did this one a while back:

https://drive.google.com/file/d/1NLAV7uhIT54JJCoS658XC9jXPgTzXgoc/view?usp=drive_link

**Comment by helge17** [comment]
There are a few other TuxGuitar icons around in svg format, all under free licenses:

https://iconduck.com/icons/276904/tuxguitar
![iconduck_tuxguitar_1](https://github.com/helge17/tuxguitar/assets/5897631/b1361ab0-9cc9-4595-888a-53ec0781e744)
![iconduck_tuxguitar_2](https://github.com/helge17/tuxguitar/assets/5897631/f48789f8-22e3-441f-9318-c2940440e999)
![iconduck_tuxguitar_3](https://github.com/helge17/tuxguitar/assets/5897631/e5190820-ad54-4446-baed-ce8439eb493c)
![iconduck_tuxguitar_4](https://github.com/helge17/tuxguitar/assets/5897631/fdca34e2-27a1-4075-9adf-7d8e54da790a)

https://www.iconarchive.com/show/papirus-apps-icons-by-papirus-team/tuxguitar-icon.html
https://www.iconarchive.com/download/i106482/papirus-team/papirus-apps/tuxguitar.svg
![iconarchive_Papirus-Team-Papirus-Apps-Tuxguitar](https://github.com/helge17/tuxguitar/assets/5897631/de1cb523-b9aa-43ec-9a1f-ff954f59ad4c)
